.gitlab/lassen-jobs.yml

```diff
@@ -105,7 +105,7 @@ clang_9_0_0_libcpp (build and test on lassen):
 
 clang_9_0_0_memleak (build and test on lassen):
   variables:
-    SPEC: "+sanitizer_tests %clang@9.0.0 cxxflags=-fsanitize=address"
+    SPEC: "+asan +sanitizer_tests %clang@9.0.0 cxxflags=-fsanitize=address"
```

Comment:
Does the datarace test also need this? (https://github.com/LLNL/Umpire/blob/7e6acc84abad6041070554c9279643a0c20db42f/.gitlab/lassen-jobs.yml#L114)

Reply:
I don't think so.  This is address sanitizer (ASAN) specific (`-fsanitize=address`)

Reply:
Mkay, just checking.

Reply:
Oh and I just looked and the datarace test runs in the CI (it fails where we already know it will), so that's a good sign lol